modules/monitoring-viewing-a-list-of-available-metrics.adoc

```diff
@@ -9,11 +9,12 @@
 As a cluster administrator or as a user with view permissions for all projects, you can view a list of metrics available in a cluster and output the list in JSON format.
 
 .Prerequisites
+* You are a cluster administrator, or you have access to the cluster as a user with the `cluster-monitoring-view` role.
 * You have installed the {product-title} CLI (`oc`).
 * You have obtained the {product-title} API route for Thanos Querier.
-* You are a cluster administrator, or you have access to the cluster as a user with the `cluster-monitoring-view` role.
+* You have logged in to your `oc` session by using a bearer token.
```

Comment:
```suggestion
* You can get bearer token via `oc whoami -t`.
```
since kube admin user can log in cluster with `oc login $<apiserver> -u kubeadmin -p $<password>`

Reply:
Updated the wording per your suggestion.